zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -156,6 +166,16 @@ pub trait GetBlockTemplateRpc {
     /// zcashd reference: [`getpeerinfo`](https://zcash.github.io/rpc/getpeerinfo.html)
     #[rpc(name = "getpeerinfo")]
     fn get_peer_info(&self) -> BoxFuture<Result<Vec<PeerInfo>>>;
+
+    /// Returns the block subsidy reward of the block at `height`,
+    /// taking into account the mining slow start and the founders reward.
+    ///
+    /// `height` can be any valid current or future height.
```

Comment:
```suggestion
    /// Returns the block subsidy reward of the block at `height` or error if `height` is less than 
    /// the height of the first halving for the current network.
    ///
    /// `height` can be any valid current or future height.
```

Reply:
Thanks, I'll do this fix in PR #6031.

Reply:
Oops, sorry, I thought this was another PR, I'll do these fixes now.